### PR TITLE
Replace react scripts with webpack

### PR DESCRIPTION
**Pull request by peterschwarz**
Replace the use of `react-scripts` with the use of a simple webpack configuration.  React scripts are very opinionated and have no real way to configure them without using the rewire library.  Likewise, it adds a
lot of complexity to the build, which makes for a slow development cycle.

This change configures webpack for production mode for `build` and `deploy`. It switches to development mode for `bundle` and `deploy-local`.  This enables debugging the saplings, even when loaded by canopy.

The `start` script is removed, as saplings (currently) do not have a way to run in isolation from canopy.

### CODE REVIEW

File: saplings/circuits/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const path = require('path');
```
Comment (dplumb94) [inline saplings/circuits/webpack.config.js:17]: Needs license header?

**Review by dplumb94 — CHANGES_REQUESTED** [review]
I am getting some issues running `npm run deploy-local`:

```ERROR in ./src/components/PlusMinusButton.js
Module not found: Error: Can't resolve 'css-loader' in '/Users/dplumb/splinter-ui/saplings/circuits'
 @ ./src/components/PlusMinusButton.js 19:0-32
 @ ./src/components/ServiceCard.js
 @ ./src/components/forms/ProposeCircuitForm.js
 @ ./src/App.js
 @ ./src/index.js```

I removed previous build artifacts, ran `npm install` and then ran `npm deploy-local`.

**Review by dplumb94 — DISMISSED** [review]

File: saplings/circuits/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        // jsx, etc
+        {
+          test: /\.js$/,
+          exclude: /node_modules/,
+          loader: 'babel-loader'
+        },
+        {
+          // styles
+          test: /\.(css|sass|scss)$/,
+          // exclude: /node_modules/,
```
Comment (dplumb94) [inline saplings/circuits/webpack.config.js]: Should this be removed or uncommented?

**Review by dplumb94 — CHANGES_REQUESTED** [review]
The debugging fixes are great!!

I'm getting this error when running locally, in the webpack dev server console (after running `npm run start` on the splinter UI.):

```
[2020-08-04T14:37:58.477Z]  "GET /sapling-dev-server/circuits/static/css/%3C!--%20Copyright%202018-
2020%20Cargill%20Incorporated%20Licensed%20under%20the%20Apache%20License,%20Version%202.0%20(the%20%2
2License%22);%20you%20may%20not%20use%20this%20file%20except%20in%20compliance%20with%20the%20License.
%20You%20may%20obtain%20a%20copy%20of%20the%20License%20at%20http://www.apache.org/licenses/LICENSE-
2.0%20Unless%20required%20by%20applicable%20law%20or%20agreed%20to%20in%20writing,%20software%20distribute
d%20under%20the%20License%20is%20distributed%20on%20an%20%22AS%20IS%22%20BASIS,%20WITHOUT%20WARR
ANTIES%20OR%20CONDITIONS%20OF%20ANY%20KIND,%20either%20express%20or%20implied.%20See%20the%20Lice
nse%20for%20the%20specific%20language%20governing%20permissions%20and%20limitations%20under%20the%20Licen
se.%20--
%3E%3Csvg%20viewBox=%220%200%2025%2026%22%20fill=%22none%22%20xmlns=%22http://www.w3.org/2000/svg%
22%3E%3Crect%20width=%221.93595%22%20height=%2217.8379%22%20transform=%22matrix(0.715643%200.698466%
20-0.707407%200.706806%2012.6187%2011.4875)%22%20fill=%22" Error (404): "Not found"
```

I also get a similar error in the splinter-ui consoles when running in Docker. They don't seem to actually be causing any issues that I can tell, but it seems to indicate some problem.

File: saplings/circuits/webpack.config.js (hunk 2 above)
Comment (peterschwarz) [reply saplings/circuits/webpack.config.js]: Should be removed - will fix.

Comment (peterschwarz) [reply saplings/circuits/webpack.config.js]: Fixed.

**Review by dplumb94 — APPROVED** [review]

**Review by dnewh — APPROVED** [review]

**Pull request merged**